source/configuration_manual/stats/openmetrics.rst

```diff
@@ -0,0 +1,74 @@
+===================================
+OpenMetrics exporter for statistics
+===================================
+
+.. versionadded:: 2.3.11
+
+Basic configuration
+===================
+
+Dovecot has support for `OpenMetrics <https://openmetrics.io/>`_ exposition for statistics.
```

Comment:
"exposition" word looked weird, but googling it I see it's a thing. But usually it's together as "exposition format", so maybe add format?